test/linear_algebra/GLA.jl

```diff
@@ -48,6 +49,28 @@ x, y = [2, 1], [7, 3, 5]
 @test antipode(dom(f))*x == -1*x
 @test adjoint(g)*y == g'*y
 
+# LinearOps instance
+#-------------------
+
+f, g = LinearOperator([0 1; 1 0]), LinearOperator([1 2 3; 4 5 6; 7 8 9])
+h, k = LinearOperator([1 -1; -1 1; 0 1]), LinearOperator([1 1; 2 2; 3 3])
+
+x, y = [2, 1], [7, 3, 5]
+@test (f⋅f)*x == f*(f*x)
+@test (f⋅h)*x == h*(f*x)
+@test (f⊕g)*[x;y] == [f*x; g*y]
+@test braid(dom(f),dom(g)) * [x;y] == [y;x]
+
+@test mcopy(dom(f))*x == [x;x]
+@test delete(dom(f))*x == []
+@test plus(dom(f))*[x;x] == 2x
+@test zero(dom(f))*Float64[] == zero(x)
+
+@test (h+k)*x == h*x + k*x
+@test scalar(dom(f),3)*x == 3*x
+@test antipode(dom(f))*x == -1*x
+@test adjoint(g)*y == g'*y
+
```

Comment:
We should add some tests that use the `functor` function to plug these maps into an expression like:
```julia
A, B = Ob(FreeLinearFunctions, :A, :B)
f, g = Hom(:f, A, B), Hom(:g, A, B)
h, k = Hom(:h, B, A), Hom(:k, B, A)
fop, gop = LinearOperator([0 1; 1 0]), LinearOperator([1 2 3; 4 5 6; 7 8 9])
hop, kop = LinearOperator([1 -1; -1 1; 0 1]), LinearOperator([1 1; 2 2; 3 3])

@test functor(compose(f,h), Dict(f=>fop, h=>hop)) * [1 1] == hop*fop*[1 1]
```